### PR TITLE
Conan 1.42 release blogpost

### PR DESCRIPTION
**Pull request by czoido**
127 0 0 1_4000_2021_11_10_New-conan-release-1-42 html

### CODE REVIEW

**Review by lasote — APPROVED** [review]
👍

**Review by memsharded — APPROVED** [review]

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+ "description": "New Conan XcodeDeps multi-config generator for Xcode, added new OSX version 12.0 (Monterey), easier aggregated environment handling.",
+ }
+</script>
+
+
+Conan 1.42 brings several significant new features. We have added a new [conan.tool.XcodeDeps
+generator](https://docs.conan.io/en/latest/reference/conanfile/tools/apple.html), which supports
+multiple configurations and easy integration with the Xcode IDE. Also, Conan now comes with OSX
+Monterrey support in its default settings. Finally, Conan now generated a script to deactivate the
+aggregated environments as it was for activating them.
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:38]: I don't understand this sentence, "as it was for activating them". Could you rephrase it?

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+
+## New XcodeDeps multi-config generator
+
+The [new XcodeDeps](https://docs.conan.io/en/latest/reference/conanfile/tools/apple.html) generator
+allows easy integration with the Xcode IDE. It supports multiple configurations, so if you install
+the packages for different *Release* and *Debug* configurations, you can switch between those two
+configurations in the IDE and the dependency information updates for the active one.
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:46]: suggestion: "...and the dependency information is updated to match the active configuration requirements"

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+Now build as usual. **Xcode** will link with the information that Conan provides for the configuration
+that's active.
+
+<p class="centered">
+    <img src="{{ site.url }}/assets/post_images/2021-11-10/4-build_project.png" align="center" alt="Build the project"/>
+</p>
+
+Besides this new generator, we have introduce support for **macOS Monterey** in the default settings that
+come with the Conan installation.
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:122]: Suggestion: "we now support **macOS Monterey** in the default Conan's settings"

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+## Deactivate aggregate environments easier 
+
+We continue making improvements in the Conan tools for managing environments. Up to this point, the
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:126]: "We continue to make improvements in the Conan tools" or "we keep improving the Conan tools"

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+script was generated for deactivation. From this version, a new `deactivate_conanbuild` or
+`deactivate_conanrun` is generated together with those files to deactivate all the environments at
+once.
+
+
+Also, please check that we have made some changes to the environment classes. Now, [Environment](https://docs.conan.io/en/latest/reference/conanfile/tools/env/environment.html) is just a generic class used by others like the [conan.tools.gnu](https://docs.conan.io/en/latest/reference/conanfile/tools/gnu.html#conan-tools-gnu) autotools helpers, and the [VirtualBuildEnv](https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualbuildenv.html#conan-tools-env-virtualbuildenv) and [VirtualRunEnv](https://docs.conan.io/en/latest/reference/conanfile/tools/env/virtualrunenv.html#conan-tools-env-virtualrunenv) generator. If you want to use it, a specialization for the current context EnvVars object needs to be obtained from it with the [vars()](https://docs.conan.io/en/latest/reference/conanfile/tools/env/environment.html#obtaining-environment-variables) method. Let's see an example:
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:133]: please check that we have made some changes -> please note that we made some changes

File: _posts/2021-11-10-New-conan-release-1-42.markdown [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+that's active.
+
+<p class="centered">
+    <img src="{{ site.url }}/assets/post_images/2021-11-10/4-build_project.png" align="center" alt="Build the project"/>
+</p>
+
+Besides this new generator, we have introduce support for **macOS Monterey** in the default settings that
+come with the Conan installation.
+
+## Deactivate aggregate environments easier
```
Comment (tapia) [inline _posts/2021-11-10-New-conan-release-1-42.markdown:124]: aggregate -> aggregated
easier -> easily
